R/modules.R

```diff
@@ -177,8 +177,8 @@ mainModuleUI <- function(id){
                                                         style = "jelly",
                                                         color = "success"),
                 verbatimTextOutput(ns("kcalMeal")),
-                plotly::plotlyOutput(ns("percentMacro")),
```

Comment:
You can restore plotly. You need to use setValue instead of setImputs in shinytests, and tests should pass.